terraform/concourse/elb.tf

```diff
@@ -47,7 +47,10 @@ resource "aws_security_group" "concourse-elb" {
     from_port       = 443
     to_port         = 443
     protocol        = "tcp"
-    cidr_blocks     = ["${split(",", var.office_cidrs)}"]
+    cidr_blocks     = [
+      "${split(",", var.office_cidrs)}",
+      "${aws_eip.concourse.public_ip}/32"
+    ]
```

Comment:
This is triggering the "Terraform diffs didn't match" bug when creating an environment from scratch (ie when the EIP is created in the same terraform run as this security group).  See https://github.com/alphagov/paas-cf/pull/123 for a workaround.